test/cmtCounterFunc.js

```diff
@@ -0,0 +1,22 @@
+import Data from './__mock__.js';
+
+export default function commentCounter(n) {
+  const data = new Data();
+  data.setData(n);
+  const comments = data.getData();
+
+  if (comments) {
+    if (comments.length <= 30) {
+      document.querySelector('#comments-number').innerHTML = `Comments (${comments.length})`;
+      return comments;
+    }
+
+    const commentLi = "<tr class='cm-li'><td colspan=3>over 30 comments</td></tr>";
+    document.querySelector('#cm-movie-existing-comment').innerHTML = commentLi;
+  } else {
+    const commentLi = "<tr class='cm-li'><td colspan=3>No comments found!</td></tr>";
+    document.querySelector('#cm-movie-existing-comment').innerHTML = commentLi;
+  }
+
+  return null;
+}
```

Comment:
- [ ] Please avoid creating new functions to test. The main idea of testing is to give you confidence that your program works as expected. However, if you're using different functions in your program and your tests, this confidence is null. Please consider testing the original functions. 🤓 🔧 


💡 _I suggest that you refactor your counters to count the DOM elements instead of the length of the array returned by the API, this would make testing easier since you don't have to handle async logic_